frontend/plugins/frontline_ui/src/modules/ticket/components/ticket-navigations/TicketNavigations.tsx

```
@@ -94,8 +94,10 @@ const Pipelines = () => {
     variables: { filter: { channelId: channelId || '' } },
   });
   useEffect(() => {
-    !pipelineId && pipelines?.[0]?._id && setPipelineId(pipelines[0]._id);
-  }, [pipelines, setPipelineId, pipelineId]);
+    channelId &&
+      pipelineId !== pipelines?.[0]?._id &&
+      setPipelineId(pipelines?.[0]?._id || null);
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(@typescript-eslint/no-unused-expressions)